steve/util.py

```diff
@@ -378,10 +378,14 @@ def out(*output, **kw):
     sys.stdout.write(output + '\n')
 
 
-def load_json_files(config):
+def load_json_files(config, path=None):
```

Comment:
Ok--I tinkered with it and I see what's going on now. My vote is nix the tri-state thing and we should just assume that `load_json_files`, `save_json_files` and `save_json_file` are saving data in the `json/` directory. Given that, we'll change the API so that they're always returning basenames and not paths relative to some unstated directory.

So the change here would be to nix the tri-state argument and always return the filename as `os.path.basename(file_name)`.

We'll have to make a minor change in `save_json_files` and `save_json_file`, too.

Does that sound ok?